infer/src/base/CommandLineOption.mli

```diff
@@ -26,19 +26,20 @@ val init_work_dir : string
     declared options.
 
     The arguments of the declaration functions are largely treated uniformly:
-    - [long] declares the option [--long]
-    - [short] declares the option [-short] as an alias
-    - [deprecated] declares the option [-key] as an alias, for each [key] in [deprecated]
-    - [default] specifies the default value
-    - [default_to_string] is used to document the default value
-    - [f] specifies a transformation to be performed on the parsed value before setting the config
-      variable
-    - [symbols] is an association list sometimes used in place of [f]
-    - [parse_mode] declares which parse mode the option is for
-    - [in_help] indicates the man pages in which the command should be documented, as generated by
-      calling infer with --help. Otherwise it appears only in --help-full.
-    - [meta] is a meta-variable naming the parsed value for documentation purposes
-    - a documentation string
+    {ul
```

Comment:
What about using the light syntax and adding an open line here?

Reply:
If a list item does not fit on the line, the heavy syntax is automatically used